recipes/perl-minion-backend-sqlite/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "perl-minion-backend-sqlite" %}
+{% set version = "v5.0.7" %}
```

Comment:
the version string should not start with a v character, as this propagates to the real version of the package.

Reply:
@bgruening 

Expected version 5.0.7 but found v5.0.7 at /mnt/sdc/khaled/miniconda3/conda-bld/perl-minion-backend-sqlite_1689685276573/test_tmp/run_test.pl line 8.